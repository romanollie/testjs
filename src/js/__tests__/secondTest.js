/**
 * @jest-environment jsdom
 */

import App from '../app';

const app = new App();

describe('Группа Тестов', () => {
    beforeAll(() => {
        app.showApp();
    });

    afterAll(() => {
        app.hideApp();
    });

    it('Тест на добавление контента на страницу в блок #app', () => {
        app.setAppHtml('some text');

        expect(app).toMatchSnapshot();
    });
});
