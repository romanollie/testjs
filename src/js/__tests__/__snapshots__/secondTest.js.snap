// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Группа Тестов Тест на добавление контента на страницу в блок #app 1`] = `
App {
  "element": <div
    id="app"
  >
    some text
  </div>,
}
`;
